step2-02/README.md

```diff
@@ -82,47 +82,49 @@ const MyComponent = () => {
 };
 ```
 
-# Layout with Stack
+## Layout with Stack
 
-Before we start, let's look at flexbox. It is really, really complex to use:
+Before we start, let's look at flexbox--a new CSS layout method which is powerful, but really, really complex to use:
 
-- a guide: https://css-tricks.com/snippets/css/a-guide-to-flexbox/
-- a tool: http://the-echoplex.net/flexyboxes/
-- did you know there were 3 or so flex box standards? i.e. Old links will have non-working code
+- A guide: https://css-tricks.com/snippets/css/a-guide-to-flexbox/
+- A tool: http://the-echoplex.net/flexyboxes/
+- Did you know there were three or so flexbox standards? (this means old articles may have non-working code)
 
-Fabric's answer is: Stack
+Fabric's answer is: Stack.
 
-> A Stack is a container-type component that abstracts the implementation of a flexbox in order to define the layout of its children components.
+**Stack** is a container-type component that abstracts the usage of flexbox to define the layout of its child components.
 
-The concepts are:
+Flexbox uses CSS styles to control:
 
 - direction
 - grow
+- shrink
 - wrap
-- shrunk
-- justify-content
+- justification
 - alignment
 
-Stack abstracts these CSS and provides a type discoverable.
+Stack abstracts these CSS styles and provides typings to make them more discoverable.
 
-Checkout a cookbook of sorts in our documentation: https://developer.microsoft.com/en-us/fabric#/components/stack
+Check out a cookbook of sorts in our [documentation](https://developer.microsoft.com/en-us/fabric#/components/stack).
 
 # Exercise
 
-1. Open up the [Documentation for DefaultButton](https://developer.microsoft.com/en-us/fabric/#/components/button)
-2. Open up the TSX files inside `components/`
-3. Replace the DOM tags with Fabric components in those TSX files with these components:
+1. Open the [documentation for DefaultButton](https://developer.microsoft.com/en-us/fabric/#/components/button)
+2. Open each TSX file inside `exercise/src/components`
+3. Replace some of the built-in HTML tags with these Fabric components:
 
 - Stack
 - DefaultButton
 - Checkbox
 - TextField
 - Pivot (for the filter)
```

Comment:
Going through the exercises, it took me a minute to figure out what this meant about using a pivot for the filter. I'm also not sure this is quite the appropriate use for a pivot--it's good appearance-wise, but just using the PivotItems as headers without any content seems a bit weird.

Reply:
it is um, weird but sometimes happens with pivots actually...